hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -161,7 +161,8 @@
             - Run with single, dual and quad SPI modes.
             - Testbench should test the following commands:
               - Read Normal, Fast Read, Fast Dual, Fast Quad, Chip Erase, Program
-
+            - Configure the SPI device to be in firmware mode and verify the testbench can dump data
```

Comment:
i dont quite follow why we need to switch into firmware mode at the end of passthrough testing.
If we are aiming to test the upload command path, it should be covered in `pass_through_collision`. 

Isn't general firmware mode testing covered in `spi_device_tx_rx`?

Reply:
This will cover `passthough_en` signal deassertion from 1 to 0, which is not covered.

Reply:
It wasn't covered when we reviewed it, later on @a-will added `chip_sw_spi_device_pass_through_collision`, which de-asserts the passthough_en when the busy bit is set.
Now it shows covered.
<img width="867" alt="Screen Shot 2022-12-08 at 10 03 28 PM" src="https://user-images.githubusercontent.com/49293026/206635841-d8062b3a-169d-4bd4-9259-55b133eb6aa1.png">

Reply:
Ok, cool. thanks for confirming @weicaiyang.

Reply:
we can probably remove this. Testing the busy bit behavior is more interesting and sufficient for this.

Reply:
Removed.